BOSCoreTechnicalWhitePaper.md

```diff
@@ -61,19 +61,37 @@ The emergence of EOS has brought a new paradigm to the blockchain. In just a few
+In addition to the abovementioned issues, there are many other potential improvements that have been actively discussed within the EOS community. As a result the BOS team believes that more experimentation should be done on EOS and more developers or teams rallied to participate in the building of the EOSIO ecosystem. Together, ecosystem participants will make eﬀorts towards application of the blockchain towards diﬀerent scenarios within different industries. BOS will be the first DPoS public chain to achieve decentralized cross-chain and the fastest consensus. It is committed to using technology to create a trusted business ecosystem and to be a DPoS public chain covering one billion users.
 
 
 # Overview 
 
-The BOS Network is committed to providing users with easy-to-access and easy-to-use blockchain services, providing a more user-friendly infrastructure for DApp operations, working to support richer application scenarios, and actively experimenting booms in DApp development. In addition to technical improvements, the BOS Network will also make attempts to improve in other areas. For example, in order to increase the level of participation of users in voting, estimator technology can be used to incentive accounts that meet certain predefined parameters. The incentives for BP (block producers) on the BOS Network will be adjusted according to the number of DApps on the chain, TPS, market value, liquidity and other indicators. Each BP incentive awarded is an encouragement for providingthe provision of more resources for the ecology. A resolution reached by a community referendumto the ecosystem. Changes to parameters on the BOS Network will be codedalgorithmically executed as much as possible, to reduce human factors inwithin the process, keeppreserve the process on chain, and maintain fairness and transparency.
-
+The BOS Network is committed to providing users with easy-to-access and easy-to-use blockchain services, providing a more user-friendly infrastructure for DApp operations, working to support richer application scenarios, and actively experimenting booms in trustless commerice.. In addition to technical improvements, the BOS Network will also make attempts to improve in other areas. For example, in order to increase the level of participation of users in voting, estimator technology can be used to incentive accounts that meet certain predefined parameters. The incentives for BP (block producers) on the BOS Network will be adjusted according to the number of DApps on the chain, TPS, market value, liquidity and other indicators. Each BP incentive awarded is an encouragement for providingthe provision of more resources for the ecology. A resolution reached by a community referendumto the ecosystem. Changes to parameters on the BOS Network will be codedalgorithmically executed as much as possible, to reduce human factors inwithin the process, keeppreserve the process on chain, and maintain fairness and transparency.
```

Comment:
>to reduce human factors inwithin the process, keeppreserve the process on chain, and maintain fairness and transparency.

to reduce human factors within the process, preserve the process on chain, and maintain fairness and transparency.